src/critic_scorer.cpp

```diff
@@ -17,19 +16,27 @@ void CriticScorer<T>::on_configure(
   loadCritics();
 }
 
-template<typename T>
-void CriticScorer<T>::getParams()
+void CriticScorer::setLoader()
```

Comment:
setLoader and configureCritics were removed from the header + `on_configure` use. The loader creation was moved to the start of `loadCritics` and configureCritics was done after the critics were pushed back onto the vector of critics.

Reply:
Done